$OpenBSD: patch-sieve_sieved_c,v 1.1 2020/08/28 09:53:04 ajacoutot Exp $

sieve/sieved.c:161:61: warning: format specifies type 'unsigned long' but the argument has type 'unsigned long long' [-Wformat]
sieve/sieved.c:176:29: warning: format specifies type 'unsigned long' but the argument has type 'uint64_t' (aka 'unsigned long long') [-Wformat]

Index: sieve/sieved.c
--- sieve/sieved.c.orig
+++ sieve/sieved.c
@@ -158,7 +158,7 @@ static void print_stringlist(const char *label, strarr
 
 static void print_time(uint64_t t)
 {
-    printf(" %02lu:%02lu:%02lu", t / 3600, (t % 3600) / 60, t % 60);
+    printf(" %02llu:%02llu:%02llu", t / 3600, (t % 3600) / 60, t % 60);
 }
 
 static void print_vallist(const char *label, arrayu64_t *list,
@@ -173,7 +173,7 @@ static void print_vallist(const char *label, arrayu64_
 
         if (!(x % 5)) printf("\n\t\t");
         if (print_cb) print_cb(i);
-        else printf(" %lu", i);
+        else printf(" %llu", i);
     }
     printf("\n\t]");
 
